control-plane/subcommand/server-acl-init/command.go

```diff
@@ -47,7 +47,7 @@ type Command struct {
 	flagInjectAuthMethodHost string
 	flagBindingRuleSelector  string
 
-	flagCreateControllerPoliciesAndBindings bool
+	flagEnableController bool
```

Comment:
This can still stay the same though. Just the name of the flag can be `-controller`

Reply:
I think we usually try to match the var name with flag name. IMO it's kind of nice to know what the flag name is by looking at the command var as you're reading through the code.